contracts/Mortal.sol

```diff
@@ -0,0 +1,10 @@
+contract Mortal {
+    /* Define variable owner of the type address */
+    address owner;
+
+    /* This function is executed at initialization and sets the owner of the contract */
+    function mortal() { owner = msg.sender; }
```

Comment:
Don't you need to run this function when initializing the new contract?

Reply:
I think this function should be private, is it private by default? If it is, then also `kill()` is?